qutebrowser/utils/utils.py

```diff
@@ -915,3 +917,32 @@ def yaml_dump(data, f=None):
         return None
     else:
         return yaml_data.decode('utf-8')
+
+
+def pastebin_version():
+    """Pastebins version and logs to messages"""
+
+    app = qutebrowser.utils.objreg.get('app')
+    http_client = httpclient.HTTPClient()
+
+    def _get_paste_title():
+        return "qute version info {}".format(qutebrowser.__version__)
+
+    @pyqtSlot(str)
+    def _on_paste_version_success(url):
+        qutebrowser.utils.message.info("Version info pastebinned"
+                                       " to: {}".format(url))
+
+    @pyqtSlot(str)
+    def _on_paste_version_err(text):
+        qutebrowser.utils.message.info("Failed to pastebin version"
+                                       " info: {}".format(text))
+
+    pbclient = pastebin.PastebinClient(http_client, parent=app,
+                                       api_url=
```

Comment:
pylint doesn't seem to like that - you  might want to break after `(` instead and indent `http_client` by four spaces (so 8 total).